### ISSUE TITLE
Manually add to current day's work time

### ISSUE THREAD

**Issue by tumbl3w33d** [Diego-Ivan/Flowtime#56]
Sometimes I forget to switch to work stage and notice after half hour. Would be great to have a (very simple) option to manually add work time for the current day, like a "+"-Button on the work stage page or something. When you're at it, the opposite might be useful as well, i.e, you forgot to switch to break but are busy with lunch for 20 min already. In such case you could press "-", remove some time and switch the stage. So far I've compensated for these cases "manually" by starting the work stage later than when I really started to work again.

**Comment by Diego-Ivan** [comment]
I'm not really sure about this feature. I think it does not make much sense in the context of the app. Nonetheless, the autostart feature may help you when you forget to start a timer after completing a work or break session.

**Comment by tumbl3w33d** [comment]
I'm using the auto start and it's great that it exists. However, it doesn't help when you don't press the switch stage button. You can sit in your break stage for "00:00" for hours without noticing.

Fortunately, Gnome has this "always on top" functionality which keeps flowtime in my view so I notice quickly. But I need to activate this always on top everyday manually. I guess, an integrated "always on top" functionality that I can activate in the preferences would solve the problem as well.

**Comment by cristiangervasi** [comment]
I agree that having the option to adjust the time of the work or break stages would be very useful. Sometimes I forget to start the timer or I get distracted by something else. Other times I don't resume working right away after the break is over. Being able to add or remove time manually would help me manage my productivity better.
